ament_cmake_google_benchmark/ament_cmake_google_benchmark/__init__.py

```diff
@@ -53,3 +80,61 @@ def main(argv=sys.argv[1:]):
             res.returncode = 1
 
     return res.returncode
+
+
+def convert_google_benchark_to_jenkins_benchmark(
+    in_file,
+    out_file,
```

Comment:
@cottsay meta safe to ignore: if this function took a `dict` and returned a `dict`, instead of dealing with files and JSON serializations, testing would be simpler.

Reply:
You're right. That's a lot cleaner.
25a94a8